### PR TITLE
109 fe UI unauthenticated pages

### LINKED ISSUE #109: FE - Make UI for unauthenticated pages

### PR DESCRIPTION
**Pull request by Itek01**
- every unauthenticated pages except blog
- made mobile menu adhere to the rest of the page
- Closes #109

### Some pages are still up for change, many components were reused. Mockup data for now
image
image
image
image
image
image
image

### CODE REVIEW

**Review by ChemsCode — APPROVED** [review]
Looks good!

File: quolance-ui/src/constants/data.ts [hunk 1]
```diff
@@ -47,17 +47,17 @@ export const headerMenu = [
       {
         id: uuidv4(),
         name: 'About US',
-        link: '/about-us',
+        link: '/why-quolance//about-us',
```
[the remark this answers is not in the document]
Comment (Itek01) [reply quolance-ui/src/constants/data.ts:50]: > Wrong url link

That's how I made the folders initially, but it was already planned that I change the UI with tailwind so I can change the foldering as well to something less confusing like 
/info-pages/ [each page] for all pages
**(that's how it is currently, I agree it's confusing)**
image

[the remark this answers is not in the document]
Comment (KA-devl) [reply quolance-ui/src/constants/data.ts:50]: ok nice

**Comment by Itek01** [comment]
@KA-devl @abdelh17 @ChemsCode 
## New css design:
image
image
image
image
image

**Comment by KA-devl** [comment]
@Itek01  some pages are black on black 
image

**Comment by KA-devl** [comment]
when background is black, either a light gray or white is good, anything else is not pleasant

**Comment by KA-devl** [comment]
image
I like the idea, however can you adjust the cards? You don't need to reuse an existing component from the project (If you have done it), you can simply create a different component and customize it on your own. Keep in mind that any reusable component that you use (The freelancer card) is subjected to change and may affect your design (You might have to adjust it later on)

**Comment by Itek01** [comment]
@KA-devl @abdelh17
Made a "basic freelancer card" component for situations like these. I also merged my branch from main already so the merge shouldn't have any conflicts. Please approve my changes when you can 🙋‍♂️
image

**Review by KA-devl — APPROVED** [review]

**Review by abdelh17 — APPROVED** [review]

**Pull request merged**